k9mail/src/main/java/com/fsck/k9/mailstore/LocalMessage.java

```diff
@@ -173,6 +175,11 @@ public void setSubject(String subject) {
         headerNeedsUpdating = true;
     }
 
+    public boolean isHighPriority() {
+        if (getHeader(highPriorityTag).length>0 && getHeader(highPriorityTag)[0].equals(xPriorityContent))
```

Comment:
this can be in `MimeMessage`, and can use `getFirstHeader` there. also, code style violations